campaigns/eocb.md

```diff
@@ -0,0 +1,79 @@
+# The Education, Outreach & Community Building Campaign
+
+Education, Outreach and Community Building
+
+## Overview
+
+Success of the Pangeo project has lead to rapid improvement of Python-based
+and related tools and technology that can dramatically improve scientific
+data analysis.  Since Pangeo is an open community effort, its continued
+success depends upon the size and knowledge base of the community, itself.
+Hence, we must build and expand the community through education, outreach,
+and related community building activities.
+
+As more scientists are trained to use the Pangeo software stack, more potential
+contributors to that software stack will arise.  It is important to teach
+scientists who are new to Pangeo and the Pangeo stack to be comfortable being
+developers, themselves.  We need to be welcoming and supportive as a community,
+so that new "users" feel comfortable being "contributors," too.  We need to
+be able to teach not only how to use the software itself, but how to program
+using best practices, including git, GitHub, and how to contribute to the
+community through common communication channels such as GitHub, Discourse, and
+through regular open community meetings.
+
+## Vision
+
+As described in the [Xdev Vision Statements](../PURPOSE.md#xdev-vision-statements),
+the main goal of the EOCB Campaign is to build a community of more than just
+*users* of the Pangeo software stack and platform.  The goal is to build a
+community of "contributors" who feel welcome to contribute at any level they
+feel comfortable.  This involves many different areas of involvement.
+
+### Education in Software Usage
+
+The first step for many in their journey into Pangeo is learning how to
+*use* the software.  Ideally this will be accomplished with both online
+educational material including videos, text-based tutorials, live
+tutorials, and demonstration Notebooks.
+
+#### Possible Projects
+
+- Add to the Notebook example gallery (https://gallery.pangeo.io)
+- Create video tutorials and/or demos
+- Live tutorials
+- Online, self-paced educational material
+
+### Education in Open Source Development
+
+Learning how to contribute to open source has a broad meaning, ranging
+from just simply reporting bugs in a GitHub issue tracker to learning how
+to write a pull request to a software package to becoming a core software
+package maintainer (including knowing how to prioretize issues, perform
+code reviews, etc.).  Users of the software should be encouraged to
+give back to the software they use in any way they feel comfortable, and
+education should be given to help users get *more* comfortable with
+contributing even more.
+
+#### Possible Projects
+
+- Live tutorials
+- Online, self-paced educational material
+- Organizing hackathons
```

Comment:
Organizing hackathons with a lot advance skill set. Since tutorials offer beginner knowledge, we should provide advance skill set to the users too. In one side, it can gain higher performance with advance skill sets, in the other side, it can make sure users using less codes achieving more functions. I thought even current documentations having too few advance materials.

Reply:
Yes.  We should say something about advanced tutorial content and training.  It seems like the *intermediate* training already exists in a number of tutorials and examples around the internet.  But beginner and advanced seem to be lacking.